fsw/src/lc_msg.h

```diff
@@ -77,8 +77,8 @@ typedef struct
 {
     CFE_MSG_CommandHeader_t CmdHeader; /**< \brief Command Header */
 
-    uint16 APNumber;   /**< \brief Which actionpoint(s) to change */
-    uint16 NewAPState; /**< \brief New actionpoint state          */
+    uint16       APNumber;   /**< \brief Which actionpoint(s) to change */
+    LC_APState_t NewAPState; /**< \brief New actionpoint state          */
```

Comment:
this should stay `uint16` because it is a message (enums are not defined length)

Reply:
Updated.
Thanks - forgot to consider that aspect.